web/src/resources/osk/kmwosk.css

```diff
@@ -490,7 +490,7 @@ div.android div.kmw-keytip-cap {
 .tablet.ios #kmw-popup-keys .kmw-key{border:none;}
 .phone.ios  #kmw-popup-keys .kmw-key{border:none;}
 
-.phone.android #kmw-popup-keys {border:none; border-radius: 2px; background-color:#ccc; padding:5px 5px 0 0;}
+.phone.android #kmw-popup-keys {border:none; border-radius: 2px; background-color:#333333; padding:5px 5px 0 0;}
```

Comment:
Longpress background styling to match the current background from
https://github.com/keymanapp/keyman/blob/31fd6ea42afef1dddd7d883fc31c5086fe9ae9b2/android/KMEA/app/src/main/res/values/colors.xml#L11